lib/platform/gitlab/index.ts

```diff
@@ -686,7 +731,7 @@ export async function setBranchStatus({
 
 export async function getIssueList(): Promise<any[]> {
   if (!config.issueList) {
-    const res = await api.get(
+    const res = await gitlabApi.getJson<{ iid: number; title: string }[]>(
```

Comment:
Create a type for this. maybe we should move all gitlab api types to `./types.ts`